File: python/properties-methods-and-instances/exercise02/main.py
# Thi import is used to import the Student class from the classStudent file
from classStudent import Student

# Instances
student1 = Student("John", 20, "Computer Science", 90.45)
student2 = Student("Jane", 21, "Computer Science", 95.20)
student3 = Student("Alice", 22, "Computer Science", 87.20)

print("Students from Universidad InterAmericana de Puerto Rico:")
print()

# Now we use the method to show the details of the student
student1.mostrarDetalles()
print("--------------------")
student2.mostrarDetalles()
print("--------------------")
student3.mostrarDetalles()

File: javascript/properties-methods-and-instances/exercise02/main.js
const Student = require('./classStudent');

// Instances
const student1 = new Student("John", 25, "Computer Science", 90);
const student2 = new Student("Jane", 22, "Web Development", 95);
const student3 = new Student("Doe", 30, "Data Science", 85);

console.log("Students from Universidad InterAmericana de Puerto Rico:");
console.log();


// mostrar los detalles de cada estudiante
student1.mostrarDetalles();
console.log("--------------------------------");
student2.mostrarDetalles();
console.log("--------------------------------");
student3.mostrarDetalles();